example/fwup-architecture/Makefile

```diff
@@ -0,0 +1,99 @@
+PROJECT = fwup-example
+BUILD_DIR = build
+
+CC=arm-none-eabi-gcc
+LD=arm-none-eabi-ld
+OCPY=arm-none-eabi-objcopy
+ODUMP=arm-none-eabi-objdump
+SZ=arm-none-eabi-size
+MKDIR=mkdir
+XXD=xxd
+
+SRCS_BOOT = boot.c
+SRCS_APP = app.c \
+		   shell/src/shell.c \
+		   app_shell_commands.c
+SRCS_LOADER = loader.c \
+			  dfu.c \
+			  shell/src/shell.c \
+			  loader_shell_commands.c \
+			  $(BUILD_DIR)/app_bin.c
```

Comment:
maybe: I'd be tempted to manually load the bin in the example using (gdb)'s load <file> <addr> I think that would be closer to real world where the binary is not baked into the loader image but rather lives at a fixed offset